src/client/components/RoomList/RoomList.css

```diff
@@ -1,25 +1,23 @@
-button {
+img {
```

Comment:
Do not style _all_ images in the app, have a better CSS selector only for this component.